ng-dev/pr/merge/merge-tool.ts

```diff
@@ -86,6 +87,24 @@ export class MergeTool {
 
     const pullRequest = await loadAndValidatePullRequest(this, prNumber, validationConfig);
 
+    if (pullRequest.validationFailures.length > 0) {
+      Log.error(`Pull request did not pass one or more validation checks. Error:`);
+
+      for (const failure of pullRequest.validationFailures) {
+        Log.error(` -> ${bold(failure.message)}`);
+      }
+      Log.info();
+      if (pullRequest.validationFailures.find((failure) => !failure.canBeForceIgnored)) {
+        Log.info(yellow(`All discovered validations are non-fatal and can be forcibly ignored.`));
+
+        if (await Prompt.confirm('Do you want to forcibly ignore these validation failures?')) {
+          return;
+        }
+      }
+
+      throw pullRequest.validationFailures[0];
```

Comment:
oh this feels incorrect. could we just throw fatal merge tool error then?

Reply:
Created new FatalPullRequestValidationFailureError class for this usage.  This also aligns better as it is now defined in the same place as the other pull request fatal errors.